### DISCUSSION TITLE
Change the tabitem header foreground

### DISCUSSION THREAD

**Discussion by MrStamina** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2986] [Q&A] [answered]
Hello, I'm trying to change the foreground header on tabItem. 

![image](https://user-images.githubusercontent.com/8642891/205082428-1d70f913-de83-4cd0-b46c-cc964914afd3.png)

I want the text in white when is not selected. I use the MaterialDesignFillTabControl style with Custom ColorZoneAssist :

```xaml
<TabControl BorderThickness="0" Grid.Row="0" SelectedIndex="{Binding SelectedIndexMainTabControl}"
                            Margin="5,10,0,0"  Style="{StaticResource MaterialDesignFilledTabControl}" materialDesign:ColorZoneAssist.Mode="Custom"
                            materialDesign:ColorZoneAssist.Foreground="#ff5a08"  
                            materialDesign:ColorZoneAssist.Background="#1f2c44"
                    Padding="5,15,0,0" Grid.ColumnSpan="3">
 <TabItem Header="Summary" >
```

What I already did : 
- Overwrite the style especially the part "This is the Header label ColorZone" line 300 in your TabControl style.
- A lot of shitty things that did not work.

I think it's not something hard but I can't see it :/
Thanks for your help

**Comment by Keboo** [comment, marked as answer]
Hi @MrStamina 

I pushed a small sample application [here](https://github.com/Keboo/MaterialDesignInXaml.Examples/tree/master/TabControl/TabControl.Colors) that shows how to do it.

The key change is here:
```xaml
<TabControl 
            Style="{StaticResource MaterialDesignFilledTabControl}"
            materialDesign:ColorZoneAssist.Mode="Custom"
            materialDesign:ColorZoneAssist.Foreground="#ff5a08"
            materialDesign:ColorZoneAssist.Background="#1f2c44">
    <TabControl.ItemContainerStyle>
        <Style TargetType="TabItem" BasedOn="{StaticResource MaterialDesignTabItem}">
            <Style.Triggers>
                <Trigger Property="IsSelected" Value="False">
                    <Setter Property="materialDesign:ColorZoneAssist.Foreground" Value="White" />
                </Trigger>
            </Style.Triggers>
        </Style>
    </TabControl.ItemContainerStyle>
    ...
</TabControl>
```

**Reply by MrStamina** [reply]
Thank you @Keboo, it works perfectly ! You saved me a lot of time :)
